### PR TITLE
[graph] Update whitelist of TPU-compatible ops

### PR DESCRIPTION
**Pull request by tayo**
Updated to reflect state as of 2018.01.03.

I am also requesting CODEOWNERS and write access on plugins folder, as I am planning to make several changes in the near future.  Changes should be localized to the TPU compatibility functionality.

### CODE REVIEW

File: tensorboard/plugins/graph/tf_graph_common/op.ts [hunk 1]
```diff
@@ -259,7 +284,10 @@ module tf.graph.op {
     // Separating these to indicate that these are manually added, as opposed to
     // those above that are gleaned from the op registry.
     'Placeholder',
-    'VarHandleOp'
+    'VarHandleOp',
+    'Merge',
```
Comment (chihuahua) [inline tensorboard/plugins/graph/tf_graph_common/op.ts]: Optional: Maybe alphabetically order this section.

File: tensorboard/plugins/graph/tf_graph_common/op.ts [hunk 2]
```diff
@@ -23,6 +23,9 @@ module tf.graph.op {
     'Acosh',
     'Add',
     'AddN',
+    'AdjustContrastv2',
```
Comment (chihuahua) [inline tensorboard/plugins/graph/tf_graph_common/op.ts]: In most cases, `V2` is capitalized, but `AdjustContrastv2` looks right based on the name of the TensorFlow kernel. Just wanted to note that. FYI :)

Comment (tayo) [reply tensorboard/plugins/graph/tf_graph_common/op.ts]: Acknowledged.

File: tensorboard/plugins/graph/tf_graph_common/op.ts (hunk 1 above)
Comment (tayo) [reply tensorboard/plugins/graph/tf_graph_common/op.ts]: Done

**Pull request merged**